src/v2/Apps/Order/Components/SavedAddressItem.tsx

```diff
@@ -48,15 +48,22 @@ export const SavedAddressItem: React.FC<SavedAddressItemProps> = (
                     {line}
                   </Text>
                   {address?.isDefault && index === 0 && (
-                    <Pill ml={0.5} variant="textSquare" hover focus>
+                    <StyledPill
+                      ml={0.5}
+                      variant="textSquare"
+                      disabled
+                      hover
+                      width={53}
+                      maxHeight={24}
```

Comment:
Not blocking but pretty important visually: In the video named "Edit modal when isDefault false", the addresses jump around a bit when the pill moves from the 2nd to the 1st -- this makes me think there's a vertical margin or maxHeight or something else that's not quite right, or maybe even some missing margin on line 1 of the address text. Could we adjust this?

Reply:
I took the height value from Figma. I think 21 will be correct.